client/requestUtil.js

```diff
@@ -196,7 +197,7 @@ RequestUtil.prototype.list = function (category, startAt, maxRecords, nextContin
     }
     // We poll from SQS
     let notificationParams = {
-      QueueUrl: `${this.SQSUrl}`,
+      QueueUrl: `${this.SQSUrlByCat[category]}`,
```

Comment:
I think we should add a check in that place does `this.SQSUrlByCat` contains a `category` item or at least if it's size is not 0. And put that info in the log to easy find if something is wrong.

Reply:
Pushed the fix according to review